src/components/VideoPlayer/BaseVideoPlayer.tsx

```diff
@@ -211,14 +209,13 @@ function BaseVideoPlayer({
                 setIsEnded(false);
             }
 
-            if (prevIsMutedRef.current && prevVolumeRef.current === 0 && !status.isMuted) {
-                updateVolume(0.25);
+            if (prevIsMuted.get() && volume.get() === 0 && !status.isMuted) {
```

Comment:
At first I thought I would be able to get rid of this part completely, however, in the full-screen mode mute states are not properly initialized on first startup, this fixes these lines now